asv/__init__.py

```diff
@@ -29,4 +29,4 @@
     if os.environ.get('__PYVENV_LAUNCHER__'):
         os.unsetenv('__PYVENV_LAUNCHER__')
 
-from . import plugin_manager
+from . import plugin_manager # noqa this dependency is not used here but it may be used in other files
```

Comment:
What's the error that you are receiving here, if this unused import is removed?

You should add the error code to ignore, not just use `noqa`

Reply:
There's no particular error as regards to this an appveyor, should I add the  flake8 error am ignoring

Reply:
If there is no error in appveyor when removing this import, why do we need to keep it and ignore the flake8 issue, instead of just removing the import?

Reply:
when I remove the line completely, appveyor fails with the build below , when I leave it with the # noqa it runs.
https://ci.appveyor.com/project/airspeed-velocity/asv/builds/42489522

Reply:
What's the relevant error in the build, can you post it here please?

Reply:
C:\projects\asv\asv\commands\__init__.py:84: KeyError
C:\projects\asv\asv\repo.py:313: UserError
these are the 2 errors in the build

Reply:
More reference below
https://paste.ofcode.org/i6iCnChMX7ewS86bVZtA5J
https://paste.ofcode.org/i6iCnChMX7ewS86bVZtA5J

all the details  are here >  https://ci.appveyor.com/project/airspeed-velocity/asv/builds/42489522